src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources.properties

```diff
@@ -295,7 +295,7 @@ MSG_Help_linux_install=\
 \          Maintainer for .deb package\n\
 \  --linux-menu-group <menu-group-name>\n\
 \          Menu group this application is placed in\n\
-\  --linux-package-deps\n\
```

Comment:
I am not familiar with this CLI option, but if it is "dependency list separated by comma and space", then it should be documented. Similar to `--add-modules`.

Also, why we need `space` as well?

For example:
```
\ --linux-package-deps <package-dep-list>\n\
\ A comma (",") separated list of required packages or capabilities for the application\n\
```

or if space is really needed:
```
\ --linux-package-deps <package-dep-list>\n\
\ A comma and space (", ") separated list of required packages or capabilities for the application\n\
```

Reply:
I mostly learned about this through trial and error. To me it seems it is just appended to the existing default dependency list, which is separated by space and comma. Haven’t tried omitting the space though and I am unsure where this is defined

Reply:
I assume "dependency list separated by comma and space" is a quote from the PR description. I understand that this is how users interpret the value of this parameter, but for jpackage this is a pass-through option. It can be any string. And it is documented accordingly:
```
Required packages or capabilities for the application
```
It doesn't say anything about the value format because it is irrelevant for jpackage.  Probably this is an overly vague description, and we can add details that the value of this parameter will be passed verbatim to rpmbuild or dpkg commands. But this is not what this PR is about.